### ISSUE TITLE
Visited style for links

### ISSUE THREAD

**Issue by ajacobs21e** [patternfly/patternfly-design#957]
Requesting a design exploration for a visited style for links. Possibly also for other elements (navigation, tabs, buttons, etc?)
@mceledonia

**Comment by ajacobs21e** [comment]
@dcaryll @coreyvickery FYI

**Comment by coreyvickery** [comment]
@ajacobs21e @mceledonia For text links, here's a list of links with a 3:1 contrast ratio with surrounding text.

https://www.w3.org/TR/2008/NOTE-WCAG20-TECHS-20081211/working-examples/G183/link-contrast.html

I don't mind a purple color (or another color on black) for visited text links as long as it passes contrast standards. As for text in other components, I have no idea, that might require a meeting with developers.

**Comment by ajacobs21e** [comment]
IBM uses purple-600

**Comment by mcarrano** [comment]
Makes sense to support this as an opt-in at a global level for standard text links, only. But would not want this to be default behavior as it might not make so much sense for product.
